scout/server/blueprints/panels/templates/panels/panel.html

```diff
@@ -104,7 +104,7 @@
           </span>
         </li>
       {% endif %}
-      {% if case and case.chanjo2_coverage %}
+      {% if case and case.chanjo2_coverage and panel.genes %}
```

Comment:
Unsure where I can see / test this one

Reply:
This will hide the link whenever you have an empty gene panel and you load the case using this panel among the panels used for the analyses, [here](https://github.com/Clinical-Genomics/scout/blob/f4205d6c2a21ca36525529c840bc9e0a74139883/scout/demo/643594.config.yaml#L142)

Can probably be tested locally by either creating a new gene panel with no genes and then loading the case with that panel, or by editing the demo gene panel by removing all its genes (to be done in the database). I can do this test, no worries!